pkg/controller/controller.go

```diff
@@ -301,6 +311,15 @@ func (c *Controller) syncHandler(key string) error {
 			return err
 		}
 
+		hasProfiles, err := k8s.ProfilesEnabled(c.kubeclientset)
+		if err != nil {
+			return err
+		}
+
+		if !hasProfiles {
```

Comment:
If this crashes when the CRD isn't present, how does the code get to the handlers for update/deploy? Would this be helpful enough on its own?